src/components/productoverview/ProductList.vue

```diff
@@ -36,6 +37,18 @@ export default {
 
   computed: {
     category: vm => vm.categories.results[0],
+
+    isLoading() {
+      return this.$apollo.loading;
+    },
+
+    passSorting() {
```

Comment:
Naming variables and methods is always hard. Especially here. Computed properties contain methods, but they are actually variables, and variables should use nouns, never verbs. Since this property contains the commercetools' sort criteria, a better wording for this could be `sortCriteria`, or `sortParameter`.